iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -312,23 +326,37 @@ TEST_F(PortManager_test, InterfacesOverflow)
             });
 
         auto interfacePointer = m_portManager->acquireInterfacePortData(iox::capro::Interfaces::INTERNAL, "itfPenguin");
-        EXPECT_THAT(interfacePointer, Eq(nullptr));
+        EXPECT_EQ(interfacePointer, nullptr);
         EXPECT_TRUE(errorHandlerCalled);
     }
+}
+
+TEST_F(PortManager_test, deleteInterfacePortfromMaximumNumberAndAddOneIsSuccessful)
```

Comment:
Nice split of the tests. Just one nitpicking, The names of all the other test cases start with a capital letter. Could you please also make it upper case.